### PR TITLE
TA#22390 Add module project_timesheet_time_control_enhanced

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

File: project_timesheet_time_control_enhanced/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
```
Comment (ddufresne) [inline project_timesheet_time_control_enhanced/__manifest__.py:4]: Codacy Issue found: [Statement seems to have no effect](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=5329619)

File: project_timesheet_time_control_enhanced/tests/test_account_analytic_line.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from ddt import ddt, data
+from odoo.tests import common
+from datetime import timedelta, date, datetime
```
Comment (ddufresne) [inline project_timesheet_time_control_enhanced/tests/test_account_analytic_line.py:6]: Codacy Issue found: [Unused timedelta imported from datetime](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=5329619)

File: project_timesheet_time_control_enhanced/models/account_analytic_line.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import pytz
+from datetime import datetime, time
+from odoo import _, api, fields, models
```
Comment (ddufresne) [inline project_timesheet_time_control_enhanced/models/account_analytic_line.py:6]: Codacy Issue found: [Unused _ imported from odoo](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=5329619)

File: project_timesheet_time_control_enhanced/models/account_analytic_line.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+from odoo.exceptions import UserError
```
Comment (ddufresne) [inline project_timesheet_time_control_enhanced/models/account_analytic_line.py:7]: Codacy Issue found: [Unused UserError imported from odoo.exceptions](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=5329619)

Comment (foutoucour) [inline project_timesheet_time_control_enhanced/models/account_analytic_line.py]:
```suggestion
```

File: project_timesheet_time_control_enhanced/models/account_analytic_line.py (hunk 3 above)
Comment (foutoucour) [inline project_timesheet_time_control_enhanced/models/account_analytic_line.py:6]:
```suggestion
from odoo import api, fields, models
```

File: project_timesheet_time_control_enhanced/tests/test_account_analytic_line.py (hunk 2 above)
Comment (foutoucour) [inline project_timesheet_time_control_enhanced/tests/test_account_analytic_line.py]:
```suggestion
from datetime import date, datetime
```

**Review by foutoucour — APPROVED** [review]

**Pull request merged**